# SNPEff will annotate the variants found in the data 

## snpeff vcf 
Create a vcf that is suitable to use for vcf 

## snpeff analysis
Load the snpeff vcf into r script 
Final spreadsheet will have numbers based on the presence of the variant in the sample. 
homozygous = 2 
heterozygous = 1 
not found = 0 
not covered = NA

## Variant type 
Extract information on genes that we are interested to look at, for example serping1 
Will get type of gene and impact of gene in genome and compare counts between groups (responder vs. nonresponder)
